projects/extension/src/background/AppMediator.ts

```diff
@@ -14,6 +13,18 @@ import {
   StateEmitter,
   SubscriptionMapping
 } from './types';
+import { SmoldotChain } from 'smoldot';
+import westend from '../../public/assets/westend.json';
+import kusama from '../../public/assets/kusama.json';
+import polkadot from '../../public/assets/polkadot.json';
+
+type RelayType = Record<string, string>
```

Comment:
Not a blocker .. but this should be a `Map<string, string>`

Reply:
This does not work:
```
Type '{ polkadot: string; kusama: string; westend: string; }' is not assignable to type 'RelayType'.
  Object literal may only specify known properties, and '"polkadot"' does not exist in type 'Map<string, string>'.ts(2322)
```